Command/MigrateCommand.php

```diff
@@ -3,57 +3,35 @@
 namespace EMS\CoreBundle\Command;
 
 use Doctrine\Bundle\DoctrineBundle\Registry;
-use Doctrine\Common\Persistence\Mapping\MappingException;
 use Doctrine\ORM\EntityManager;
-use Doctrine\ORM\OptimisticLockException;
-use Doctrine\ORM\ORMException;
 use Elasticsearch\Client;
-use EMS\CoreBundle\Elasticsearch\Bulker;
 use EMS\CoreBundle\Entity\ContentType;
-use EMS\CoreBundle\Entity\Revision;
 use EMS\CoreBundle\Exception\CantBeFinalizedException;
 use EMS\CoreBundle\Exception\NotLockedException;
-use EMS\CoreBundle\Form\Form\RevisionType;
-use EMS\CoreBundle\Repository\ContentTypeRepository;
-use EMS\CoreBundle\Repository\RevisionRepository;
-use EMS\CoreBundle\Service\DataService;
-use EMS\CoreBundle\Service\Mapping;
+use EMS\CoreBundle\Service\ImportService;
```

Comment:
To clarify the move of the command to MakerBundle, the ImportService should stay in CoreBundle!
However this should probably be named the DocumentService --> which is not a good name, but better then import.

Reply:
60332743f49ed4023617f67d740abf6e80937810